src/python/espressomd/constraints.py

```diff
@@ -290,8 +289,7 @@ def field_from_fn(cls, box_size, grid_spacing, f, codim=None):
 
     @classmethod
     def field_coordinates(cls, box_size, grid_spacing):
-        """Returns an array of the coordinates of  the grid
-        points required.
+        """Returns an array of the coordinates of the grid points required.
```

Comment:
Why is this not 

"""
Returns an array of the coordinates of the grid points required.

Reply:
Writing the brief on the same line as the triple quotes is not forbidden by the numpydoc style guide: https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard

Reply:
No, sure, was just wondering as you changed that in other places.

Reply:
Right, I have done that in 2 or 3 places, either the single line was longer than 80 chars, or it was followed by an indented paragraph. The latter is really annoying, because inconsistent line indentation in a docstring will be interpreted by Sphinx as a definition list, causing the text to be formatted differently depending on the browser. In that case I introduce a newline after the triple quote and align everything until the definition list disappears from the HTML output.